applications/sofa/gui/qt/CMakeLists.txt

```diff
@@ -294,12 +294,8 @@ if(NOT SOFA_NO_OPENGL)
     endif()
 endif()
 
-if(NODEEDITOR_FOUND)
-    target_link_libraries(${PROJECT_NAME} PUBLIC ${NODEEDITOR_LIBRARY})
-    if (SOFA_BUILD_RELEASE_PACKAGE OR CMAKE_SYSTEM_NAME STREQUAL Windows)
-        sofa_install_libraries(PATHS ${NODEEDITOR_LIBRARY})
-    endif()
-    target_include_directories(${PROJECT_NAME} PUBLIC "$<BUILD_INTERFACE:${NODEEDITOR_INCLUDE_PATH}>")
+if(SOFAGUIQT_ENABLE_NODEGRAPH)
+    target_link_libraries(${PROJECT_NAME} PUBLIC nodes)
```

Comment:
NodeEditor::nodes

Reply:
@guparan it works with NodeEditor::nodes. I corrected the PR, I let you mark it as ready if you don't object